eng/versioning/external_dependencies.txt

```diff
@@ -7,14 +7,14 @@
 
 # Format;
 # groupId:artifactId;dependency-version
-com.fasterxml.jackson.core:jackson-annotations;2.12.5
-com.fasterxml.jackson.core:jackson-core;2.12.5
-com.fasterxml.jackson.core:jackson-databind;2.12.5
-com.fasterxml.jackson.dataformat:jackson-dataformat-avro;2.12.5
-com.fasterxml.jackson.dataformat:jackson-dataformat-xml;2.12.5
-com.fasterxml.jackson.datatype:jackson-datatype-jdk8;2.12.5
-com.fasterxml.jackson.datatype:jackson-datatype-jsr310;2.12.5
-com.fasterxml.jackson.module:jackson-module-parameter-names;2.12.5
+com.fasterxml.jackson.core:jackson-annotations;2.13.0
+com.fasterxml.jackson.core:jackson-core;2.13.0
+com.fasterxml.jackson.core:jackson-databind;2.13.0
+com.fasterxml.jackson.dataformat:jackson-dataformat-avro;2.13.0
+com.fasterxml.jackson.dataformat:jackson-dataformat-xml;2.13.0
```

Comment:
@Netyyyy jackson-dataformat-xml is changed to 2.13.0 here, isn't it?

Reply:
yes ,in external_dependencies.txt , it write 2.13.0. But in actually use ,such as module ``azure-resourcemanager-appplatform`` ,it's dependency ``azure-resourcemanager-resources``
![image](https://user-images.githubusercontent.com/92105726/143516168-dc3a8396-b4ce-4f7e-b6aa-95f19bd881e6.png)
in this latest version , it controls ``azure-core`` which version is 1.22.0
![image](https://user-images.githubusercontent.com/92105726/143516352-51cde4a2-7b43-40a4-81ba-65233ee8207b.png)
and in this version ,
![image](https://user-images.githubusercontent.com/92105726/143516449-a58dff43-a3fa-4169-805d-744fceaf9514.png)
so that's the truely version , i think.

Reply:
I've done my lecture. Told you TestFromSource uses the latest beta as dependencies.